data/courses.yaml

```
@@ -158,3 +158,10 @@
     but would now say that re-framing the goal as formalising something new in a format that is useful to you as a learner
     (e.g. as a worksheet/interactive part of a textbook) is a better goal pedagogically
     (and could still lead to, e.g., a mathlib contribution in the process even if that isn't the explicit goal).
+- name: Logic and Modelling
+  instructor: Rob Lewis, Jasmin Blanchette, Anne Baanen
+  location: Vrije Universiteit Amsterdam
+  website: https://studiegids.vu.nl/en/2022-2023/courses/X_401015
+  material: https://leanprover.github.io/logic_and_proof/logic_and_proof.pdf
+  tags: ['introduction to logic', 'modal logic']
+  summary: This course introduces computer science students to propositional logic, predicate logic and modal logic. The course uses two formal proof systems: natural deduction (exam) and Lean (homework assignments).
```

Is this the offending line?
```suggestion
  summary: >
    This course introduces computer science students to propositional logic, predicate logic and modal logic.
    The course uses two formal proof systems: natural deduction (exam) and Lean (homework assignments).
```